client/src/App.js

```diff
@@ -1,30 +1,25 @@
-import { useEffect, useState } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { useMediaQuery } from 'react-responsive';
+import Nav from './Components/Nav';
+import AppContext from './Components/AppContext';
+import RecordForm from './Components/RecordForm';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 function App() {
-  const [serverData, setServerData] = useState("");
+  const isDesktopOrLaptop = useMediaQuery({
+    query: '(min-width: 900px)'
+  });
 
-  useEffect(() => {
-    async function getServerData() {
-      const resp = await fetch('/api/hello');
-      const data = await resp.json();
-
-      console.log('Data from server:', data);
-
-      setServerData(data.message);
-    }
-
-    getServerData();
-  }, []);
+  const appState = { isDesktopOrLaptop }
```

Comment:
What data type is`isDesktopOrLaptop`? It looks like you are using it as a boolean, but if the value is `true`, which is it -- desktop or laptop? It would be better to name it "isLaptop" so it's more clear what a true and false value mean. 

Or "isDisplayLaptop" if you want it more obvious it's about the display size.

Or `isDesktopOrLaptop` with values of `'desktop'` and `'laptop'`.